plutus-tx/src/PlutusTx/Builtins.hs

```diff
@@ -202,6 +215,276 @@ verifySchnorrSecp256k1Signature
 verifySchnorrSecp256k1Signature vk msg sig =
   fromBuiltin (BI.verifySchnorrSecp256k1Signature vk msg sig)
 
+-- | Converts an 'Integer' into its 'BuiltinByteString' representation.
+--
+-- = Notes
+--
+-- Throughout, let @maxInteger@ be the maximum function for 'Integer's,
+-- and @absInteger@ be the absolute value function for 'Integer's.
+-- We define @zeroes :: 'Integer' -> 'BuiltinByteString'@ be the
+-- function which, given input @i@, produces a 'BuiltinByteString' @bs@ such
+-- that:
+--
+-- * @'lengthByteString' bs@ @=@ @'maxInteger' 0 i@; and
```

Comment:
```suggestion
-- * @'lengthOfByteString' bs@ @=@ @'maxInteger' 0 i@; and
```

throughout

Reply:
You might want to have a look at the rendered haddock, that's how I spotted the linking issues.